Gemfile

```diff
@@ -2,3 +2,4 @@
 source "https://rubygems.org"
 
 gemspec
+gem "fastlane"
```

Comment:
This should not be added here, but in the .gemspec file as development dependencies.